packages/controls/src/form/TForm/TForm.js

```diff
@@ -74,7 +82,15 @@ function RenderForm(args) {
 			return !isEmpty(warnings);
 		},
 		fieldError(fieldName) {
-			return !!(touched[fieldName] && warnings[fieldName]);
+			// I'm not sure of the proper implementation of this.
+			// I'm Passing in an array to find the proper error item from the object array.
+			// Using it in /home/jonathan/Desktop/thr4New/src/lib/AddDynamicFormInput.tsx line 67
```

Comment:
The local path here is kinda useless.

To me it looks like this AddDynamicFormInput component should probably be a part of thr-addons. Then it would make sense to have these changes here.

Essentially we want to be able to support `fieldName` that is an array as well as the default string, right?

Reply:
The path was temporary.

And yes to the other two.

Reply:
Go ahead and add AddDynamicFormInput to thr-addons then. We may as well move it here while we are doing a release of @thx/controls anyways.